ui/app/mainui/AppMain.qml

```diff
@@ -215,13 +216,27 @@ Item {
         }
     }
 
+    Settings {
+        id: appMainLocalSettings
```

Comment:
We probably won't be doing a migration from the QML Settings to the DB settings when we end up working on syncing the settings between devices. This is mostly an FYI, that while it's nice to have it being saved in the Settings for now, users will probably have to do it again later.

The other option is to just hide the checkbox for now.